README.rst

```diff
@@ -355,6 +357,11 @@ See the `html5lib documentation`_ for further information.
 .. _html5lib: https://pypi.python.org/pypi/html5lib
 .. _html5lib documentation: https://code.google.com/p/html5lib/wiki/UserDocumentation#Sanitizing_Tokenizer
 
+Search
+------
+
+djangocms-text-ckeditor works well along `aldryn-search <https://github.com/aldryn/aldryn-search>`_ to enable text content to be searchable using Haystack.
```

Comment:
``djangocms-text-ckeditor works well with `aldryn-search <https://github.com/aldryn/aldryn-search>`_ to make text content using Haystack``